testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/partialexport/PartialExportTest.java

```diff
@@ -115,6 +127,26 @@ public void testExport() {
         checkSecretsAreMasked(rep);
     }
 
+    private void checkServiceAccountRoles(UserRepresentation serviceAccount, boolean rolesExpected) {
+        Assert.assertTrue("User is a service account", serviceAccount.getUsername().startsWith(ServiceAccountConstants.SERVICE_ACCOUNT_USER_PREFIX));
+        Assert.assertNull("Password should be null", serviceAccount.getCredentials());
+        if (rolesExpected) {
+            List<String> realmRoles = serviceAccount.getRealmRoles();
+            Assert.assertEquals(3, realmRoles.size());
+            Assert.assertTrue(realmRoles.containsAll(Arrays.asList("uma_authorization", "user", "offline_access")));
+
+            Map<String, List<String>> clientRoles = serviceAccount.getClientRoles();
+            Assert.assertNotNull(clientRoles.get("test-app-service-account"));
+            Assert.assertEquals(2, clientRoles.get("test-app-service-account").size());
+            Assert.assertTrue(clientRoles.get("test-app-service-account").containsAll(Arrays.asList("test-app-service-account", "test-app-service-account-parent")));
```

Comment:
Just a suggestion how to write this in a more simple way. It can stay like it is if you want.
```suggestion
            assertThat(clientRoles.get("test-app-service-account"), containsInAnyOrder("test-app-service-account", "test-app-service-account-parent"));
```

This approach could be applied to other asserts as well in this method.

Reply:
okis, perfect. As soon as I have clear the other two points I'll do this one.

Reply:
Done!